### ISSUE TITLE
Delay between process starts or restarts?

### ISSUE THREAD

**Issue by aib** [Supervisor/supervisor#1189]
My processes are failing immediately after startup.

Supervisor is trying to restart them indefinitely, which is good, but it's doing this at the fastest pace it can, which is bad. My CPU usage is 100% and my supervisor logs are flooded.

Is there any way to add a delay before starting or restarting a program?

**Comment by mnaberez** [comment]
Duplicate of #487
Duplicate of #323
